08_book_titles/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_writer :title
+
+  def title
+    exc = ['the', 'a', 'an', 'and', 'or', 'in', 'the', 'of']
```

Comment:
This can be extracted to a constant to avoid allocating the array each time this method is called.